src/wallet/rpcdump.cpp

```diff
@@ -1151,13 +1152,18 @@ UniValue importmulti(const JSONRPCRequest& mainRequest)
 
     //Default options
     bool fRescan = true;
+    bool fUpdate = true;
```

Comment:
New code should be snake_case, without the prefix

Reply:
Should we also rename the other variables on this method?

Reply:
Just new code.